Update scim.md (#7581)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/manage-access/scim.md

## Manage user licenses with SCIM

You can manage user license assignments via SCIM with a user attribute in your IdP environment. This ensures accurate license assignment as users are provisioned in the IdP and onboarded into your dbt account.
:::note

>     Analyst licenses are only available for certain plans. Assigning an Analyst license via SCIM will result in a user update error if that license type is not available in your dbt account. 

Reviewer: ```suggestion

    The `Analyst` license is only available for select plans. Assigning an `Analyst` license via SCIM will result in a user update error if that license type is not available for your dbt account. 
```

## Manage user licenses with SCIM

You can manage user license assignments via SCIM with a user attribute in your IdP environment. This ensures accurate license assignment as users are provisioned in the IdP and onboarded into your dbt account.

> :::note

Reviewer: ```suggestion
:::note Analyst license assignment
```
